src/components/atoms/Logo/styles.tsx

```diff
@@ -1,13 +1,18 @@
-import styled from "styled-components"
-import { colors, fsize, fweight } from "../../../styles/vars"
+import styled from "styled-components";
+import { colors, fsize, fweight } from "../../../styles/vars";
+import { media } from "../../../styles/breakpoints";
 
 
 export const StyleLogoContainer = styled.div``;
 
-export const StyleLogo = styled.text`
+export const StyleLogo = styled.p`
```

Comment:
Idk if you're already implemented this but most of the time the logo is wrapped in <a> (in this case a <Link>) tag. 
In a navbar, when you click the logo it should always redirect you to the home page.

Reply:
Fixed. Thanks!